CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Zilliqa Coding and Review Guidelines
+
+1. All submitted **Pull Requests** are assumed to be ready for review.  
+   If not, they must be marked with explicit  `[in progress]` or have "[WIP]" in title.
+
+2. - **Pull requests** which fix a bug should have branch name starting with `fix/`
```

Comment:
:bulb: Branch naming styles are somewhat problematic. There is no way to resolve this naming issue without closing a pull request and then opening a new one. I would typically recommend omitting this guideline, and only enforcing it internally (i.e. for users who push branches to the root repository as opposed to creating branches in a fork).

Reply:
@KaustubhShamshery obviously you don't have to take my suggestion. I'm just pointing out some observations based on my experiences on GitHub over the past several years. 😄